frontend/src/components/FormControl/FormLabel/FormLabel.tsx

```diff
@@ -81,6 +87,14 @@ export const FormLabel = ({
       flexDir="column"
       overflowWrap="break-word"
       {...labelProps}
+      sx={{
+        ...(isMRFPrefilled && {
```

Comment:
/nit for acronyms, we usually only capitalise the first of each acronym set i.e.,  `MRFHTTPSQL` vs `mrfHttpSql`.